src/reseeding.rs

```diff
@@ -109,9 +155,9 @@ impl<S, R: SeedableRng<S>, Rsdr: Reseeder<R>> SeedableRng<(Rsdr, S)> for
 }
 
 /// Something that can be used to reseed an RNG via `ReseedingRng`.
-pub trait Reseeder<R: ?Sized>: Debug {
+pub trait Reseeder<R: ?Sized> {
```

Comment:
Why lose `Debug`?

Reply:
I think [this API guideline](https://rust-lang-nursery.github.io/api-guidelines/future-proofing.html#data-structures-do-not-duplicate-derived-trait-bounds-c-struct-bounds) was appropriate. Besides I see no functional need for the `Debug` bound here.